config/projectlist_1_0_0.xsd

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" ?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
```

Comment:
Once the schema has matured, it makes sense to define an own namespace for it. This will make it easier, if we want to upgrade it (e.g. version 2.0.0). But that's not needed right now.